### DISCUSSION TITLE
添加在线播放功能

### DISCUSSION THREAD

**Discussion by rong6** [Meloong-Git/PCL#200] [提问] [answered]
爬取网抑云音乐等的音乐资源播放！！！

**Comment by Big-Cake-jpg** [comment, marked as answer]
PCL2 不是媒体播放器，是 Minecraft 启动器，该功能不会被制作的

**Reply by rong6** [reply]
可是PCL2不就是mod下载器+整合包下载器+音乐播放器吗（bushi）

**Reply by Big-Cake-jpg** [reply]
你一个启动器要什么在线播放音乐......那不是网易云音乐的客户端就在那里摆着呢（

而且你还想让这个仓库被 DMCA Takedown 吗

**Comment by tou002** [comment]
pcl2是我的世界启动器只可以播放本地音乐
